pkg/tnf/handlers/command/command.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+)
+
+// Command provides a Command test implemented using command line tool Command.
+type Command struct {
+	result  int
+	timeout time.Duration
+	args    []string
+	Output  string
+	// adding special parameters
+}
+
+const (
+	ocCommand = "oc get %s -n %s -o json -l %s"
```

Comment:
Shouldn't the handler named OcGetCommand if "oc get" is hardcoded? If it meant to be generic, the command part should become an arg as well.